### PR TITLE
Add attributes to services GET api

### PR DESCRIPTION
**Pull request by ZaidYounan**
Related PR: https://github.com/easy2use/gearbox/pull/3541

### CODE REVIEW

**Comment by ZaidYounan** [comment]
@Gotenks-J Does this need to be merged? The [associated PR](https://github.com/easy2use/gearbox/pull/3541) was merged a while ago

File: endpoints/services.md [hunk 1]
```diff
@@ -77,6 +78,18 @@ Authorization: Bearer $ACCESS_TOKEN
...
+      "tolerance_days": 123,
+      "interval_kms": 123,
+      "interval_hours": 123,
+      "interval_days": 123,
+      "actual_interval_kms": 123,
+      "actual_interval_hours": 6,
+      "actual_interval_days": 123,
+      "kms_ontime_status_id": 321,
+      "hours_ontime_status_id": 654,
+      "days_ontime_status_id": 987,
```
Comment (Gotenks-J) [inline endpoints/services.md]: These should be:
```
"kms_ontime_status": "ontime",
"hours_ontime_status": "overdue",
"days_ontime_status": "no_prior_service",
```

As we don't return an integer for these fields

File: endpoints/services.md [hunk 2]
```diff
@@ -68,6 +68,7 @@ Authorization: Bearer $ACCESS_TOKEN
       "service_type": "B",
       "service_type_alias": "Oil and Filters",
       "service_number": 123,
+      "fault_num": 123,
```
Comment (Gotenks-J) [inline endpoints/services.md]: This should be a string

Comment (ZaidYounan) [reply endpoints/services.md]: Good catch! These issues should be resolved now

**Pull request merged**